AGENTS.md

```
@@ -265,14 +255,14 @@ Use a strict `tsconfig.json` suitable for browser builds:
 - **Immutability first**: Prefer `const`, `readonly`, and `Readonly<T>`; avoid
   mutating props or inputs.
 - **Functions**: Extract meaningfully named helpers when a function grows long.
-  Keep trivial functions on one line when readability allows:
-  ```ts
-  export const mkId = (n: number): Id => new Id(n);
-  ```
+    Keep trivial functions on one line when readability allows:
+
+    ```ts
+    export const mkId = (n: number): Id => new Id(n);
+    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)